.changeset/moody-dancers-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@oriflame/backstage-plugin-score-card': major
```

Comment:
```suggestion
'@oriflame/backstage-plugin-score-card': minor
```
Now we have 0.7.0, this will bump the version to 0.8.0.
Any reason you wan to bump the package to 1.0?